docs/primary/topic_based_expanded_reference/GUIDE.md

```diff
@@ -1352,30 +1352,6 @@ reminder for future uses of stack:
 * stack.yaml can override the flag settings for individual packages
 * stack.yaml specifies which packages to include
 
-## Comparison to other tools
```

Comment:
I was just trying to break things down where possible into smaller "topic" type files that could potentially be referenced from multiple places, e.g., "Installation and Upgrading" could've been another potential "topic".